canonical-web-and-design/ubuntu.com
# Remove form background submit

## Done
- Removed the JS to background submit forms
- Removed the JS for the whitepaper which was removed earlier
- Added global form handling for generic success

## QA
- Open the demo
- Check a traditional form like /core/contact-us and check the form submits and goes to the thank you page
- Go to the something like /internet-of-things click contact to show the modal form submit test data and see it returns to the iot page with a success message
- Try a white paper download form at the /download/server, check they work.

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/4437
